pho/src/pages/space.tsx

```diff
@@ -36,13 +41,38 @@ const SpacePage: React.FC<{}> = () => {
     void dispatch(fetchSpaceDetail(Number(id)));
   }, []);
 
+  const [isMusicMuted, setIsMusicMuted] = useState(true);
+  const [isVideoVisible, setIsVideoVisible] = useState(false);
+
   return (
     <div>
-      <Heading>
-        Space #{id} - {name}
-      </Heading>
-      <p>#Bg-Video-Component</p>
-      <p>Socket ID: {socket.id}</p>
+      <div
+        className="location-name-box"
+        style={isVideoVisible ? {} : { display: 'none' }}
+      >
+        <div className="location-text">{name}</div>
+      </div>
+
+      <div
+        style={isVideoVisible ? { position: 'absolute' } : { display: 'none' }}
+      >
+        <Pomodoro
+          timestamp={new Date(startTime ?? '').getTime()}
```

Comment:
@richardtlp please help me take a look at this part. thankiu

Reply:
It's OK for now. Let me check the Pomodoro component implementation again with @thienanhvu.